### DISCUSSION TITLE
Command Line Changes

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1032] [Ideas] [answered]
Possibly change `Command Line` feature key mapped from `escape` to `ctrl/cmd + shift + p` or just add it to make Monkeytype feel a bit more like Visual Studio Code (possibly also call it command palette)

**Comment by ObiKeahloa** [comment]
Instead of changing the  thing he could make it both.

**Reply by unrealapex** [reply]
TBH, the name command line doesn't fit it's purpose as you aren't "running" any commands, rather just enabling,disabling, or modifying settings. Also you can't use the command line in full screen as if you try to press Escape, it exits full screen as you close the command line(reason for rebinding the key).

**Comment by Miodec** [comment, marked as answer]
Added #1731
